web/src/hooks/queries/useDisputeTemplate.ts

```diff
@@ -40,10 +45,29 @@ export const useDisputeTemplate = (disputeID?: string, arbitrableAddress?: `0x${
             true
           );
           console.log("useDisputeTemplate:", disputeTemplate);
+          const disputeTemplateInput = disputeTemplate?.templateData;
+          const dataMappingsInput = disputeTemplate?.templateDataMappings;
 
-          return disputeTemplate;
+          configureSDK({ apiKey: alchemyApiKey });
```

Comment:
I believe we don't need to call `configureSDK` since it's called already inside the actions that need it (i.e. https://github.com/kleros/kleros-v2/blob/dev/kleros-sdk/src/dataMappings/actions/callAction.ts#L7)